examples/hosts.yaml

```diff
@@ -8,6 +8,7 @@
 #   - Hosts must be specified in a group's hosts:
 #     and they must be a key (: terminated)
 #   - groups can have children, hosts and vars keys
+#   - a group hostlist is definable anywhere a group is. multilple hostlists for a group should be merged
```

Comment:
I would rephrase:

- groups are unique and global, so defining a group in multiple locations aggregates all the data to the global name 

as its not just the hostlist, its vars and child group relationships